src/components/parameters.js

```diff
@@ -89,7 +89,7 @@ const Parameters = ({ showParameters, hideParameters }) => {
                 <Grid item xs={6}>
```

Comment:
Why we do not use the same values for `xs` as for `study app `?

Reply:
I agree. Values changed to be the same as the ones used in study app.